webapp/accounts/models.py

```diff
@@ -14,8 +14,18 @@
 from django.core.mail import mail_admins
 from mailing.helpers import get_template_by_id, render_mail_template, mail_user
 from constance import config
+from django.contrib.auth.models import Group
 
 
+# add email adress to Group #
```

Comment:
Waarom is dit conditioneel, kan het veld niet aan de `Group` class worden toegevoed?

Reply:
Group is een standaard class van Django. Als ik hem subclass, dan pakt het admin gedeelte dat niet op. Op deze manier wel.
De conditie had ik gebruikt omdat ik aannam dat dat best practice is

Reply:
Uiteraard sta ik open voor suggesties of een andere aanpak

Reply:
Ik begrijp het niet helemaal, je subclasst `Group` toch al, bij de definitie van het proxy-model? Waarom kan dat velt niet direct in dat proxy-model worden gezet?

Reply:
Een proxy-model kan niet op de normale wijze extend worden. Dit zou je kunnen zien als een work around. Een andere manier is een normale subclass te maken van Group. Dan is het tricky om de admin site overal de juiste Group subclass te laten gebruiken. Zal ik dat laatste doen? Vanuit het model gezien voelt tweede optie netter, maar maakt andere delen complexer. Heb jij een suggestie of een andere oplosrichting?

Reply:
Als het op de huidige manier werkt vind ik het prima! Ik had deze aanpak gewoon nog nooit eerder gezien.

Weet je overigens zeker dat het werkt? Volgens de migration lijkt het erop dat er een proxy-model wordt gemaakt zonder extra velden (zie https://github.com/rikva/voko/pull/151/files#diff-9b5c154af7abcf940f45a6df9039337d0f541c5152522a921968da7b1d65105eR19).

Nogmaals, als de huidige aanpak werkt dan vind ik het prima om te mergen 👍

Reply:
Goed gezien. Dat had ik niet door. Blijkbaar komen de auth migrations in de pip env. terecht. Ook de eerdere auth migrations staan niet in Git. Hoe ga jij normaal om met wijzigingen in het auth model? Je draait geen 'makemigrations', maar alleen 'migrate' tijdens deployment neem ik aan? Of is dit in het verleden anders gegaan?

Er is wel een manier om ze binnen het project te laten genereren, door MIGRATION_MODULES variabele te definieren voor auth:auth/migrations in base.py. Maar als ik dat nu toevoeg, moeten we de eerder gegenereerde migratie files van productie inchecken, anders zijn de dependencies stuk. Ik weet niet of mijn lokaal gegeneerde auth migraties nl 1-op-1 kloppen met huidige productie

Reply:
>Hoe ga jij normaal om met wijzigingen in het auth model?

Dat heb ik eigenlijk nog nooit nodig gehad. Het is niet heel gebruikelijk om de ingebouwde modellen van Django te wijzigen. 

Ik snap dat het voor deze feature handig is om een mail field toe te voegen aan het Group model. Maar in principe is het bedoel als een verzameling van permissions.

Ik zie eigenlijk ook niet waar dit nieuwe veld gebruikt zou worden. Ik zie dat je deze template tag hebt toegevoegd: `{{ account.userprofile.contact_person.email_address }}` - dat werkt toch al zonder model wijzigingen?